.travis.yml

```diff
@@ -5,9 +5,9 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-- 2.6.5
 - 2.7.5
 - 3.0.3
+- 3.1.2
```

Comment:
Since Chef 18 embeds ruby 3.1.0, we should match that here. That way, any rspec tests will run under the same ruby version that the full chef-client run will use.